setup.cfg

```diff
@@ -48,7 +50,7 @@ test =
     %(style)s
     pytest >=5.3
     pytest-cov
-    peakdet
+    peakdet>=0.5.0
```

Comment:
With the physutils integration, do we still need peakdet?

Reply:
yes, there is a unit test with peak finding